crates/cargo-bsp/src/project_model/workspace.rs

```diff
@@ -70,13 +68,17 @@ impl ProjectWorkspace {
     }
 
     // Cargo metadata is called with `--all-features`, so we can get all features because
-    // we want the output to contain all the packages - even those feature-dependent
+    // we want the output to contain all the packages - even those feature-dependent.
+    // `--filter-platform` flag filters the output based on the target platform which allows obtaining
+    // more specific information about project's dependencies. The flag is used for `Rust Workspace`
+    // Request.
     pub fn call_cargo_metadata_command(
         project_manifest_path: &PathBuf,
         filter_platform: bool,
     ) -> Result<cargo_metadata::Metadata, Error> {
-        let filter_platform_options = if filter_platform {
```

Comment:
to szczerze, ja już chyba wolę poprzednią opcję. Unless ta Ci się bardziej podoba xd